src/test/java/xyz/vegaone/easytracking/userstory/UserStoryServiceTest.java

```diff
@@ -39,9 +39,7 @@
     @Autowired
     private TaskService taskService;
 
-    @Test
-    public void createUserStoryTest(){
-        //given
+    private UserStory buildAndSaveUserStory(){
```

Comment:
The same comment as for the Bug comment in the other PR. You can safely save the userstory inside the private method and return the saved object.